addons/account/models/account_move_send.py

```diff
@@ -301,8 +301,10 @@ def _is_applicable_to_company(self, method, company):
         return True
 
     @api.model
-    def _is_applicable_to_move(self, method, move):
+    def _is_applicable_to_move(self, method, move, **context):
         """ TO OVERRIDE - """
+        if method == 'email':
+            return bool(context.get('mail_partner_ids')) or bool(move.partner_id.commercial_partner_id.email)
```

Comment:
Is this not enough?
```suggestion
            return bool(context.get('mail_partner_ids'))
```
By default, is not `move.partner_id.commercial_partner_id.email` already inside `mail_partner_ids` ?
By consistency with the previous code and the stable policy, wouldn't be safer to do something like
```suggestion
        if method == 'email' and 'mail_partner_ids' in context:
            return bool(context['mail_partner_ids'])
```

Reply:
We can if I add it when computing the summary in batch sending too (and it will actually be fully correct instead of a lazy approximation) :+1: